src/executor.rs

```diff
@@ -253,6 +275,10 @@ impl CommandExt for Executor {
     fn output_checked_with(&mut self, succeeded: impl Fn(&Output) -> Result<(), ()>) -> Result<Output> {
```

Comment:
Can we comment on `ExecutorChild::Wet` that it represents both wet and damp outputs?